lib/node_modules/@stdlib/math/base/special/minmaxabsf/README.md

```diff
@@ -0,0 +1,258 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# minmaxabsf
+
+> Return the minimum and maximum absolute single-precision floating-point numbers.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var minmaxabsf = require( '@stdlib/math/base/special/minmaxabsf' );
+```
+
+#### minmaxabsf( x,y )
+
+Returns the minimum and maximum absolute single-precision floating-point values in a single pass.
+
+```javascript
+var v = minmaxabsf( 4.0, 3.0 );
+// returns <Float32Array>[ 3.0, 4.0 ]
+
+v = minmaxabsf( +0.0, -0.0 );
+// returns <Float32Array>[ 0.0, 0.0 ]
+```
+
+If any argument is `NaN`, the function returns `Nan` for both the minimum value and the maximum value.
+
+```javascript
+var v = minmaxabsf( 4.2, NaN );
+// returns <Float32Array>[ NaN, NaN ]
+
+v = minmaxabsf( NaN, 3.14 );
+// returns <Float32Array>[ NaN, NaN ]
+```
+
+#### minmaxabsf.assign( x, y, out, stride, offset )
+
+Returns the minimum and maximum absolute single-precision values in a single pass and assigns the results to a provided output array.
+
+```javascript
+var Float32Array = require( '@stdlib/array/float32' );
+
+var out = new Float32Array( 2 );
+
+var v = minmaxabsf.assign( 5.0, -1.0, out, 1, 0 );
+// returns <Float32Array>[ 1.0, 5.0 ]
+
+var bool = ( v === out );
+// returns true
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var randu = require( '@stdlib/random/base/randu' );
```

Comment:
We can refactor this example to use `random/array/uniform` and `console/log-each-map`.